gui/default/index.html

```diff
@@ -806,8 +806,8 @@ <h4 class="panel-title">
                         <th><span class="fas fa-fw fa-desktop"></span>&nbsp;<span translate>Remote GUI</span></th>
                         <td class="text-right" ng-attr-title="Port {{deviceCfg.remoteGUIPort}}">
                           <!-- Apply RFC6874 encoding for IPv6 link-local zone identifier -->
-                          <a ng-if="idToRemoteGUI[deviceCfg.deviceID]" href="{{idToRemoteGUI[deviceCfg.deviceID].replace('%', '%25')}}">{{idToRemoteGUI[deviceCfg.deviceID]}}</a>
-                          <span ng-if="!idToRemoteGUI[deviceCfg.deviceID]">Unreachable</span>
+                          <a ng-if="hasDeviceGUIAddr(deviceCfg)" href="{{deviceGUIAddr(deviceCfg).replace('%', '%25')}}">{{deviceGUIAddr(deviceCfg)}}</a>
+                          <span ng-if="!hasDeviceGUIAddr(deviceCfg)">Unreachable</span>
```

Comment:
If we don't do any probing, I don't think that "Unreachable" is a sensible information.  If you mean to indicate there is no suitable connection to construct an address from, maybe we should change the wording. Something like "No connection"?

And I thought the discussion turned out in favour of the "remote GUI" naming, also for variables / functions?

Reply:
Perhaps "Unknown"?

Reply:
Actually we do know.  Either there is no connection, or it's known not to be related to the host itself, but a relay server. So "No connection" is still my favorite, for the second case "No direct connection" would be more accurate.

Reply:
Well, there is a remote device and it has a GUI, we just don't know how to connect to it. We also use it for other things like unknown folder status so it's already there and translated. "No [direct] connection" would be more like the underlying reason we don't know the GUI address.

Reply:
But the heading "Remote GUI" is not about an address.  The fact is that we do not know how to (hopefully) reach it, which is somewhere between "Unreachable" and "Unknown".  Maybe get more explicit and say "Address unknown"?